live/.env.example

```diff
@@ -1,3 +1,5 @@
 API_BASE_URL="http://api:8000"
-REDIS_HOST="localhost"
-REDIS_PORT="6379"
\ No newline at end of file
+REDIS_URL="redis://localhost:6379"
+PORT=3003
```

Comment:
remove port from env example.

Reply:
Done and added a comment to reduce confusion of repeated variables 🙌